packages/node_modules/@webex/internal-plugin-llm/src/llm.ts

```diff
@@ -0,0 +1,114 @@
+// @ts-ignore
+import Mercury from '@webex/internal-plugin-mercury';
+
+
+import {LLM} from './constants';
+
+export const config = {
```

Comment:
i dont see this getting used

Reply:
we use this while registering for the internal plugin